tests/test_flow.py

```diff
@@ -3448,7 +3489,9 @@ def test_csv_source_with_none_values():
         "2021-04-21 15:56:53.385444",
     ]
     assert termination_result[1].key == "b"
-    assert termination_result[1].body == ["b", True, None, math.nan, math.nan, None]
+    excepted_result = ["b", True, math.nan, math.nan, math.nan, math.nan]
+    for x, y in zip(termination_result[1].body, excepted_result):
+        assert (isinstance(x, float) and isinstance(y, float) and math.isnan(x) and math.isnan(y)) or x == y
```

Comment:
I think it's better split to an if and two separate asserts, because if this fails, all you'll see is that the condition was false.